main.tex

```diff
@@ -595,6 +595,39 @@ \section*{Discussion}
 %%
 
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Cross-resistance between pyrethroids and DDT}
+
+
+%%
+%
+The VGSC protein is the physiological target of both pyrethroid insecticides and DDT @@REF.
+%
+The \texttt{L995F} and \texttt{L995S} alleles are known to increase resistance to both of these insecticide classes @@REF.
+%
+Except in a few locations, DDT has not been used for IRS within the last two decades, and is not suitable for use in bed-nets @@REF.
+%
+DDT was, however, used in Africa for several pilot IRS projects carried out during the first global campaign to eradicate malaria, during the 1950s and 1960s @@REF.
+%
+DDT was also used in agriculture throughout @@decades, and although agricultural usage has greatly diminished since @@decade, some usage may remain.
+%
+In this study we reported evidence of positive selection on haplotypes carrying \texttt{L995F} and haplotypes carrying \texttt{L995S}, as well as haplotypes carrying the \texttt{I1527T+V402L} combination.
+%
+We also found 14 other non-synonymous substitutions within \textit{Vgsc} that have arisen in association with \texttt{L995F} and appear to be positively selected.
+%
+Given that pyrethroids have dominated public health insecticide use for two decades, it is reasonable to assume that the selection pressure on these alleles is primarily due to pyrethroids.
+%
+If a resistance allele had been selected primarily by the earlier phase of DDT usage, then we might expect to see a faster rate of EHH decay for haplotypes carrying that allele relative to other alleles selected by recent pyrethroid usage.
+%
+However, we found no significant difference between EHH decay for haplotypes carrying \texttt{L995F} or \texttt{L995S} @@TODO confirm.
+%
+EHH decay was faster on the upstream flank for haplotypes carrying \texttt{I1527T}, however this may be affected by selection for the combination of \texttt{I1527T+V402L} and thus for recombination events that bring these two variants together on the same haplotype.
+%
+Nevertheless, it is possible that some if not all of the alleles we have reported provide some level of cross-resistance to DDT as well as pyrethroids, and that earlier DDT usage may have contributed at least in part to their selection.
```

Comment:
Maybe "...and that earlier and/or contemporary DDT usage..."?

Reply:
I think this is a pretty safe way of bringing up the subject of DDT, when I get a chance I will try to dig out the papers that show the differing effects of S and F with type 1 and 2 pyrethroids and DDT (it would be a good point to mention the different pyrethroids too).

Reply:
Very good idea to discuss different pyrethroids.